api/tests/unit/segments/test_unit_segments_views.py

```diff
@@ -1556,3 +1557,51 @@ def test_include_feature_specific_query_filter__false(
     # Then
     assert response.json()["count"] == 1
     assert [res["id"] for res in response.json()["results"]] == [segment.id]
+
+
+def test_clone_segment(
```

Comment:
```suggestion
def test_clone_endpoint__uses_segment_clone_service(
```

Adds more meaning. 😉